sln/OData.E2E.sln

```diff
@@ -166,6 +166,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Spatial.Tests", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.OData.PublicApi.Tests", "..\test\PublicApiTests\Microsoft.OData.PublicApi.Tests.csproj", "{D482D182-ABFA-41FF-B4E2-67ED968CEE99}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.OData.Client.E2E.TestCommon", "..\test\EndToEndTests\Common\Microsoft.OData.Client.E2E.TestCommon\Microsoft.OData.Client.E2E.TestCommon.csproj", "{73DB460C-D581-4E2F-B692-A9508CE8C9AD}"

```

Comment:
Can you explain the changes made to this file? Also, I would assume we would be creating a new solution

Reply:
When setting up the `WebApplicationFactory<T>' we need to set-up the `UseContentRoot`. This setting takes in the path to the `.sln` file. The way the project has been set-up the .sln file is in a different folder from the tests folder. This leads to exceptions being thrown. Having the .sln file in the base folder seems to solve this problem. I'm not sure whether there is a better way of doing this.

Reply:
We'll create a new solution for the tests? I used the current set-up where the tests are included in the other projects solution.

Reply:
What I mean was, a new solution that includes the new tests and the source projects that the tests reference.

Reply:
Added a new solution